README.md

```diff
@@ -1 +1,157 @@
+  <br/>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leadboard Project] <a name="about-project"></a>
+
+> This is a project that is using API call for showing which player have the highest score. (Under construction)
+
+## 🛠 Built With <a name="built-with"></a>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JS</a></li>
+    <li><a href="https://reactjs.org/">webPack</a></li>
+  </ul>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+- **[Adding New Task]**
+- **[Deleting Task]**
+- **[Editing Task]**
+- **[Clear all the completed tasks at once]**
+- **[Fully Created with ES6]**
+- **[Fullty tested with jest testing framework]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo">https://hidar37.github.io/To-Do-List/dist/</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Node.js
+- Browser (any)
+- Working PC
+
+### Setup
+
+- In order to Clone this repository to your PC please follow the below steps:
+ > copy this link [https://github.com/Hidar37/To-Do-List.git]
+ > Open your (cmd) or (git bash) and type [cd desktop] to change the directore and selec the desktop
+ > go to your (cmd) or (git bash) and type this commad [git clone <copy the repo link>] the repo will be save to your desktop.
+
+### Usage
+
+To run the project, execute the following command:
+
+> Go to that downloaded folder and open the index.html file and your good to go.
+
+### Run tests
+> 5 Test cases have been used 
+> In CMD you can type [ npm test ] in order to see the test cases result.
+
+### Deployment
+
+You can deploy this project using:
+- Github page 
+- Hosting in a web server
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors">Haider Ahamd & Fahim Khan</a>
+
+> Mention all of the collaborators of this project.
+👤 **Author1**
+
+- GitHub: [@Hidar37](https://github.com/Hidar37)
+
+👤 **Author2**
+
+- GitHub: [@Fahim-Khan-P](https://github.com/Fahim-Khan-P)
+
```

Comment:
- [OPTIONAL] Kindly remove the name(s) of  the student(s) who are not collaborator(s) on this project.